source/_integrations/solarlog.markdown

```diff
@@ -9,7 +9,8 @@ ha_codeowners:
   - '@Ernst79'
 ha_domain: solarlog
 ha_platforms:
-  - sensor
+  - Sensor
+  - Energy
```

Comment:
These are platforms, not categories. I think you mean to edit `ha_category` instead?

Reply:
Yes, sorry, I will change it